editor/src/clj/editor/gl/vertex2.clj

```diff
@@ -14,29 +14,19 @@
 
 (ns editor.gl.vertex2
   (:require [clojure.string :as str]
+            [editor.buffers :as buffers]
             [editor.gl :as gl]
             [editor.gl.protocols :refer [GlBind]]
             [editor.gl.shader :as shader]
             [editor.protobuf :as protobuf]
-            [editor.scene-cache :as scene-cache]
-            [internal.util :as util])
-  (:import [com.jogamp.common.nio Buffers]
+            [editor.scene-cache :as scene-cache])
+  (:import [clojure.lang Counted]
+           [com.jogamp.common.nio Buffers]
            [com.jogamp.opengl GL GL2]
-           [java.nio Buffer ByteBuffer ByteOrder DoubleBuffer FloatBuffer IntBuffer LongBuffer ShortBuffer]))
+           [java.nio Buffer ByteBuffer IntBuffer]))
 
 (set! *warn-on-reflection* true)
-
-(defn type-size
-  ^long [type]
-  (case type
-    :ubyte  Buffers/SIZEOF_BYTE
-    :byte   Buffers/SIZEOF_BYTE
-    :ushort Buffers/SIZEOF_SHORT
-    :short  Buffers/SIZEOF_SHORT
-    :uint   Buffers/SIZEOF_INT
-    :int    Buffers/SIZEOF_INT
-    :float  Buffers/SIZEOF_FLOAT
-    :double Buffers/SIZEOF_DOUBLE))
+(set! *unchecked-math* :warn-on-boxed)
```

Comment:
* Moved some stuff around between the `vertex2.clj`, `graphics.clj`, and `buffers.clj` modules.

All this stuff is in `buffers.clj` now.